### PR TITLE
travis: update go version

### PR DESCRIPTION
**Pull request by dvrkps**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ os:
 
 language: go
 go:
-  - 1.8
+  - 1.x
```
Comment (zippy) [inline .travis.yml]: holochain requires at least version 1.8, which is why I set this that way.  Is there a way to specify some version or above?

Comment (dvrkps) [reply .travis.yml]: @zippy i added 1.8.x and 1.9.x instead 1.x

**Comment by dvrkps** [comment]
@zippy ping :)
happy holidays.

**Comment by philipbeadle** [comment]
Looks ok as its just the Go version change.  Normally through dev and the test coverage has to go up :)

**Pull request merged**